components/discord_webhook/actions/send-message-advanced/send-message-advanced.js

```diff
@@ -39,9 +39,28 @@ module.exports = {
         "avatarURL",
       ],
     },
+    include_sent_via_pipedream_flag: {
+      type: "boolean",
+      optional: true,
+      default: true,
+      label: "Include link to workflow",
+      description: "Defaults to `true`, includes a link to the workflow at the end of your Discord message.",
+    },
   },
   async run() {
-    const content = this.message;
+    let content = this.message;
+
+    let link = `https://pipedream.com/@/${process.env.PIPEDREAM_WORKFLOW_ID}`;
```

Comment:
these are new env vars that Giao added yesteday, to populate the workflow URL and the line below will append the specific event ID, so you can go directly to the specific event that triggered the message

Reply:
oh but actually I realized right now these env vars are only exposed to the alpha environment, which you don't have access to (yet). so don't be alarmed when those come back as `undefined`.